assets/js/src/core/modules/app/nav/main-nav.tsx

```diff
@@ -177,31 +177,31 @@ export const MainNav = (): React.JSX.Element => {
                   <ul className={ 'main-nav__list-inline' }>
                     <li><IconTextButton
                       icon={ { value: 'pimcore' } }
-                        >Default</IconTextButton></li>
+                        >{t('navigation.default')}</IconTextButton></li>
                     <li><IconTextButton
                       icon={ { value: 'cdp' } }
                       type={ 'default' }
-                        >CDP</IconTextButton></li>
+                        >{t('navigation.cdp')}</IconTextButton></li>
```

Comment:
We don't need to translate the perspectives (as this is only a static example prepared in the template. It depends on the user configuration which perspectives we will have.

We need to translate the main nav entries itself (like "Tools", "Notes&Events" etc.). Yes, in the module Calling the `t()` method will result in an error, but we can just pass the label as translation key there and then call the `t()` function in the template.